packages/simcore-sdk/src/simcore_sdk/node_ports_common/dbmanager.py

```diff
@@ -49,7 +49,7 @@ async def _get_node_from_db(
 @tenacity.retry(**PostgresRetryPolicyUponInitialization().kwargs)
 async def wait_till_postgres_responsive(dsn: DataSourceName) -> None:
     if not is_postgres_responsive(dsn):
-        raise Exception
+        raise Exception(f"Could not reach Postgres on {dsn.to_uri()}")

```

Comment:
Can you please change that generic exception to a node port one? we should not use generic exceptions.